share/functions/vi_increment_decrement.fish

```diff
@@ -0,0 +1,34 @@
+alias vi_dec 'vi_increment_decrement subtract'
+alias vi_inc 'vi_increment_decrement add'
```

Comment:
Are these aliases needed to implement the functionality? I am not sure they really should be there, given that they'd probably be user visible.